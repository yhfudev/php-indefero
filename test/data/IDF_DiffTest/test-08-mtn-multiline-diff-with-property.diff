#
# old_revision [9b51b47c6fdd4918f970b81a8672503b8bb30402]
#
# patch "a"
#  from [b3d40c241ccfd6ff1a06047a9f6d609227d2893f]
#    to [4ba8cb789fc34bd4557b5bbf7cb9f4b68b3a9f5e]
# 
#   set "a"
#  attr "foo"
# value "bar"
#
============================================================
--- a	b3d40c241ccfd6ff1a06047a9f6d609227d2893f
+++ a	4ba8cb789fc34bd4557b5bbf7cb9f4b68b3a9f5e
@@ -1,2 +1,3 @@ ls
 ls
 ls
+l
