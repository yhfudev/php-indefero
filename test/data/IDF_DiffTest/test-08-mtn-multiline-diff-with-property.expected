<?php return array (
  'a' => 
  array (
    'chunks' => 
    array (
      0 => 
      array (
        0 => 
        array (
          0 => '1',
          1 => '1',
          2 => "ls\r\n",
        ),
        1 => 
        array (
          0 => 2,
          1 => 2,
          2 => "ls\r\n",
        ),
        2 => 
        array (
          0 => '',
          1 => 3,
          2 => "l\r\n",
        ),
      ),
    ),
    'chunks_def' => 
    array (
      0 => 
      array (
        0 => 
        array (
          0 => '1',
          1 => '2',
        ),
        1 => 
        array (
          0 => '1',
          1 => '3',
        ),
      ),
    ),
  ),
);